specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -60,12 +60,6 @@ union ProvisioningState {
 
   @doc("Update in progress.")
   Updating: "Updating",
-
```

Comment:
Note: Delete properties to cause breaking changes, and automation pipeline will block this PR merge action and add `BreakingChange-Javascript-Sdk` for us